### ISSUE TITLE
incompatibility with native-base

### ISSUE THREAD

**Issue by r3wt** [benhurott/react-native-masked-text#78]
Hi thanks for your cool library. It seems right up my alley.

I am attempting to use this in my view but nothing seems to be happening. I am using the `customTextInput` option with an `Input` from `native-base`. the following is my code:

```js
<Item floatingLabel>
    <Label>Birthday</Label>
    <TextInputMask
        customTextInput={Input}
        type={'datetime'}
        options={{
            format:'MM/DD/YYYY'
        }}
        {...this.model('birthday')}
    />
</Item>
```
`this.model()` returns an object of shape `{value,onChangeText}`. When i type into the input a date no mask seems to be applied, for example when typing `10101995` i would expect `10/10/1995` to be displayed, but it isn't. it appears no mask is being applied. 

I am not quite sure what is wrong with my approach here, any ideas?

**Comment by benhurott** [comment]
Hi,

I'm investigating, but it seems to be some incompatibility with `native-base`.
I think native-base changes the default behavior of child text-inputs.

Let me do some more research.

Thanks for reporting.

**Comment by benhurott** [comment]
Yes, it is a behavior for `floating label`.

![image](https://user-images.githubusercontent.com/3187772/41503691-91caf0d8-71b0-11e8-860b-296d7bf9a01d.png)

I will try some workaround and send you the code =).

**Comment by benhurott** [comment]
Use this sample code =):

```jsx
import React from 'react';
import { Container, Header, Content, Form, Item, Label, Input } from 'native-base';

import { MaskService } from 'react-native-masked-text'

export default class App extends React.Component {
  constructor(props) {
    super(props)

    this.state = {
      birthday: ''
    }
  }
  render() {
    return (
      <Container>
        <Header />
        <Content>
          <Form>
            <Item floatingLabel>
              <Label>Birthday</Label>
              <Input
                returnKeyType="next"
                clearButtonMode="always"
                autoCapitalize="none"
                autoCorrect={false}
                value={this.state.birthday}
                keyboardType={'numeric'}
                onChangeText={(text) => this.setState({
                  birthday: MaskService.toMask(
                    'datetime',
                    text,
                    {
                      format: 'MM/DD/YYYY'
                    })
                })} />
            </Item>
          </Form>
        </Content>
      </Container>
    );
  }
}
```

**Comment by r3wt** [comment]
Dude, thank you so much. im a react-native noob still, and i appreciate you taking time to help me.

On Jun 16, 2018 8:17 PM, Ben-hur Santos Ott <notifications@github.com> wrote:

Closed #78<https://github.com/benhurott/react-native-masked-text/issues/78>.

**Comment by r3wt** [comment]
It works perfectly! I owe you a coffee or a beer!
